### PR TITLE
Change pvc creation time to 4.2 supported values

### PR DESCRIPTION
**Pull request by keesturam**

### CODE REVIEW

File: tests/e2e/test_pvc_creation_performance.py [hunk 1]
```diff
@@ -102,15 +100,14 @@ def test_multiple_pvc_creation_measurement_performance(
         if total_time > 60:
```
Comment (Shrivaibavi) [inline tests/e2e/test_pvc_creation_performance.py]: is it not total_time > 180 ?

Comment (keesturam) [reply tests/e2e/test_pvc_creation_performance.py]: Yes, fixed it.

**Comment by keesturam** [comment]
This PR tries to cover multiple issues. 
1) Update new timeout values
2) Fix for #720 
3) Add params with multiple volume sizes for pvc creation perf test
4) Fix for #1229 

I'm going to break this down into multiple PRs as follows and close this PR. 

separate PR has been sent for 1 & 3 -> https://github.com/red-hat-storage/ocs-ci/pull/1237
Fix for #720 is covered as part of #1235 
A Separate PR will be sent for address #1229

**Comment by keesturam** [comment]
#1235 and #1237 is merged now. I'll send a separate PR to address #1229 

closing this PR.

**Pull request closed without merging**